### DISCUSSION TITLE
Collegarsi a seconda casa

### DISCUSSION THREAD

**Discussion by millenniumy2k** [indomus/forum#1381] [HUB: Home Assistant]
è possibile "collegarsi" ad una seconda casa e vedere le prese, luci, ecc che ho li?
Penso che la soluzione sia con una vpn, il problema ora che ho la stessa classe ovunque, 192.168.1.xx...

**Comment by 2Max2Pax** [comment]
Ma nella seconda casa hai un secondo hub?

**Reply by millenniumy2k** [reply]
no, altrimenti era facile :-)
Dovrei comunque aver trovato la soluzione, purtroppo devo cambiare classe di IP

**Reply by sky7176** [reply]
Ciao io collego casa, dove ho Hassio, con l'ufficio mediante una vpn net to net, io ho due firewall , uno a casa e l'altro in ufficio.
In questo modo gestisco i dispositivi che ho in ufficio da homeassistant di casa.

Giuseppe

**Reply by snakuzzo** [reply]
@millenniumy2k mi sa che c'è un custom component, ma secondo me, se sei pratico, potresti sfruttare le [API REST ](https://developers.home-assistant.io/docs/api/rest/)messe a disposizione da Home Assistant per creare le entità di un HA sull'altro...
